.circleci/config.yml

```diff
@@ -74,12 +74,14 @@ jobs:
       - checkout
       - go/mod-download-cached
       - setup_remote_docker:
-          version: 17.07.0-ce
+          version: 18.06.0-ce
       - attach_workspace:
           at: .
-      - run: ln -s .build/linux-amd64/thanos thanos
-      - run: make docker
-      - run: docker run thanos --help
+      # register qemu to support mulit-arch
+      - run: docker run --privileged linuxkit/binfmt:v0.7
+      - run: make docker-build
+      - run: make docker-test
+      - run: make docker-manifest
```

Comment:
Another thing: `docker manifest push ...` needs to happen _after_ pushing the Docker containers to a registry:
https://github.com/prometheus/circleci/blob/master/src/commands/publish_images.yml#L47-L48
https://github.com/docker/cli/issues/2396
https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/